diff --git a/node_modules/@liamcottle/rustplus.js/rustplus.proto b/node_modules/@liamcottle/rustplus.js/rustplus.proto
index d893865..89b4d93 100644
--- a/node_modules/@liamcottle/rustplus.js/rustplus.proto
+++ b/node_modules/@liamcottle/rustplus.js/rustplus.proto
@@ -229,7 +229,7 @@ message AppInfo {
 	required uint32 wipeTime = 6;
 	required uint32 players = 7;
 	required uint32 maxPlayers = 8;
-	required uint32 queuedPlayers = 9;
+	optional uint32 queuedPlayers = 9;
 	optional uint32 seed = 10;
 	optional uint32 salt = 11;
 	optional string logoImage = 12;
@@ -276,12 +276,12 @@ message AppEntityPayload {
 	message Item {
 		required int32 itemId = 1;
 		required int32 quantity = 2;
-		required bool itemIsBlueprint = 3;
+		optional bool itemIsBlueprint = 3;
 	}
 }
 
 message AppTeamInfo {
-	required uint64 leaderSteamId = 1;
+	optional uint64 leaderSteamId = 1;
 	repeated AppTeamInfo.Member members = 2;
 	repeated AppTeamInfo.Note mapNotes = 3;
 	repeated AppTeamInfo.Note leaderMapNotes = 4;
@@ -291,14 +291,14 @@ message AppTeamInfo {
 		required string name = 2;
 		required float x = 3;
 		required float y = 4;
-		required bool isOnline = 5;
-		required uint32 spawnTime = 6;
-		required bool isAlive = 7;
-		required uint32 deathTime = 8;
+		optional bool isOnline = 5;
+		optional uint32 spawnTime = 6;
+		optional bool isAlive = 7;
+		optional uint32 deathTime = 8;
 	}
 
 	message Note {
-		required int32 type = 2;
+		optional int32 type = 2;
 		required float x = 3;
 		required float y = 4;
 	}
@@ -336,9 +336,9 @@ message AppMarker {
 		required int32 quantity = 2;
 		required int32 currencyId = 3;
 		required int32 costPerItem = 4;
-		required int32 amountInStock = 5;
-		required bool itemIsBlueprint = 6;
-		required bool currencyIsBlueprint = 7;
+		optional int32 amountInStock = 5;
+		optional bool itemIsBlueprint = 6;
+		optional bool currencyIsBlueprint = 7;
 		optional float itemCondition = 8;
 		optional float itemConditionMax = 9;
 	}
